pkg/kontext/dockerignore_test.go

```diff
@@ -0,0 +1,210 @@
+package kontext
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestDirectoryHasDockerIgnore(t *testing.T) {
```

Comment:
I feel like a lot of the tests in this file could generally be consolidated as a Go "table test": https://github.com/golang/go/wiki/TableDrivenTests